placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -145,10 +171,39 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
                 }
             }
         });
-
+        if(clearEnabled)
+            enableClearButton(true);
         super.setDropDownBackgroundResource(R.drawable.pacv_popup_background_white);
     }
 
+    private void enableClearButton(boolean value){
+        if(!value) {
+            this.setCompoundDrawables(null, null, null, null);
+            return;
+        }
+        if(imgClearButton == null)
+            imgClearButton = getResources().getDrawable(R.drawable.ic_clear_black_24dp);
```

Comment:
given that this is a vector asset, we'll need backward compat inflation so as not to affect older platforms

Reply:
Fixed using ContextCompat.getDrawable(context, R.drawable.ic_clear_black_24dp);

Reply:
Fixed